libqtile/backend/base/__init__.py

```diff
@@ -0,0 +1,2 @@
+from libqtile.backend.base.core import Core  # noqa: F401
```

Comment:
I don't know that I love this move, it mostly just amounts to more cd-ing around when trying to reason about the code. Is there a technical reason to need this base class package?

Reply:
A base `window.py` file will also be added, so this was just to keep them together. We could instead just have the base files within base/ and not a nested class. Also happy to do it that way!